fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,32 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
+
+    printf("AAA collabclient_notifySessionJoining() cc: %p, fv:%p\n", cc, fv);
```

Comment:
@frank-trampe, since `cc` wasn't even used in `collabclient_notifySessionJoining`, I removed the check on `cc` but added it to this debug statement